MAUI/Autocomplete/Getting-Started.md

```diff
@@ -0,0 +1,235 @@
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:editors="clr-namespace:Syncfusion.Maui.Inputs;assembly=Syncfusion.Maui.Inputs"
+             xmlns:local="clr-namespace:AutocompleteSample"             
+             x:Class="AutocompleteSample.MainPage">
+
+       <ContentPage.BindingContext>
+            <local:SocialMediaViewModel />
+       </ContentPage.BindingContext>
+
+       <ContentPage.Content>
+            <!--Setting ItemsSource-->
+            <editors:SfAutocomplete x:Name="autocomplete" 
+                                    Width="250"
+                                    ItemsSource="{Binding SocialMedias}" />
+        </ContentPage.Content>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SocialMediaViewModel socialMediaViewModel = new SocialMediaViewModel();
+autocomplete.BindingContext = socialMediaViewModel;
+autocomplete.ItemsSource = SocialMediaViewModel.SocialMedias;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> Set the 'SocialMediaViewModel' instance as the `BindingContext` of your control; this is done to bind properties of 'SocialMediaViewModel' to `Autocomplete`.
+
+**Step 3:** Set the `TextMemberPath` and `DisplayMemberPath`.
+
+The `Autocomplete` control is populated with the list of social media. However, because the 'SocialMedia' model contains two properties 'Name' and 'ID', it is necessary to intimate, which property should be a display value in the selection box portion and drop-down suggestion of the `Autocomplete` control.
+
+`TextMemberPath` - This property path is used to get the value for displaying in the selection box portion of the `Autocomplete` control when an item is selected. The default value is `String.Empty`.
+
+`DisplayMemberPath` - This property path is used to specify the name or path of the property displayed for each data item in the drop-down list. The default value is `String.Empty`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfAutocomplete x:Name="autocomplete" 
+                        DisplayMemberPath = "Name"
+                        TextMemberPath = "Name"
+                        ItemsSource="{Binding SocialMedias}" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+autocomplete.DisplayMemberPath = "Name";
+autocomplete.TextMemberPath = "Name";
+
+{% endhighlight %}
+{% endtabs %}
+
+## Selection
+
+The `Autocomplete` allows the user to select an item from the drop-down list by clicking the `Enter` key or losing focus from the text box.
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfAutocomplete x:Name="autocomplete"
+                        DisplayMemberPath = "Name"
+                        TextMemberPath = "Name"
+                        ItemsSource="{Binding SocialMedias}" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+autocomplete.DisplayMemberPath = "Name";
+autocomplete.TextMemberPath = "Name";
+
+{% endhighlight %}
+{% endtabs %}
+
+The following gif image illustrates the result of the above code.
```

Comment:
The following gif image illustrates the result of the above code:

Reply:
Changes are made.